Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean

```diff
@@ -229,4 +229,18 @@ theorem cos_le_one_div_sqrt_sq_add_one {x : ℝ} (hx1 : -(3 * π / 2) ≤ x) (hx
   · simp
   · exact (cos_lt_one_div_sqrt_sq_add_one hx1 hx2 hx3).le
 
+theorem norm_exp_I_mul_ofReal_sub_one_le {x : ℝ} : ‖.exp (.I * x) - (1 : ℂ)‖ ≤ ‖x‖ := by
+  rw [Complex.norm_exp_I_mul_ofReal_sub_one]
+  calc
+    _ = 2 * |Real.sin (x / 2)| := by simp
+    _ ≤ 2 * |x / 2| := (mul_le_mul_iff_of_pos_left zero_lt_two).mpr Real.abs_sin_le_abs
+    _ = _ := by rw [abs_div, Nat.abs_ofNat, Real.norm_eq_abs]; ring
```

Comment:
I think perhaps extracting `lemma abs_mul_sin_div_le_abs (r) : |r * sin (x / r)| ≤ |x| :=` here is worthwhile?

Reply:
On second thoughts, that's maybe not so useful. Perhaps @grunweg has opinions.

Reply:
I'm not convinced that that lemma is generally useful. Maybe, but I think it's fine without the extraction.